Havtorn/Source/Engine/Core/MathTypes/Vector.h

```diff
@@ -374,7 +374,7 @@ namespace Havtorn
 
 	inline std::string SVector::ToString() const
 	{
-		char buffer[32];
+		char buffer[64];
```

Comment:
why 64?

Reply:
"{X: , Y: , Z: }" => 15 chars, Leaves 49 chars for float values X, Y & Z to be pasted into %.1f.
With 49 chars to share for X, Y & Z each should get 16 chars. Which allows values up to 13 digits.
Ran into an issue where the buffer was overrun with 32 as the buffer size.